sklearn/feature_selection/tests/test_rfe.py

```diff
@@ -275,7 +307,13 @@ def test_rfecv_grid_scores_size():
 
         score_len = np.ceil(
             (X.shape[1] - min_features_to_select) / step) + 1
+
+        # TODO: Remove in v1.2 when grid_scores_ is removed
         assert len(rfecv.grid_scores_) == score_len
```

Comment:
same remark here.

Reply:
Thank you! I have replaced them with `pytest.warns`.